data/events/2024/nairobi/main.yml

```
@@ -132,3 +132,334 @@ sponsor_levels:
     label: Media
   - id: community
     label: Community
+program:
+  - title: "Registration, Breakfast, Sponsors"
+    type: custom
+    date: 2024-06-01T08:00:00+03:00
+    start_time: "08:00"
+    end_time: "09:00"
+  - title: "Opening Welcome"
+    type: custom
+    date: 2024-06-01T08:00:00+03:00
+    start_time: "09:00"
+    end_time: "09:15"
+  - title: "Samuel Kadima"
+    type: talk
+    date: 2024-06-01T08:00:00+03:00
+    start_time: "09:15"
+    end_time: "09:45"
+  - title: "Sponsors"
+    type: custom
+    date: 2024-06-01T08:00:00+03:00
+    start_time: "09:45"
+    end_time: "09:55"
+  - title: "Ian Koome"
```

`title` refers to a filename, so in this case, it would be

```suggestion
  - title: "ian-koome"
```

This is true for all of your talk slots, so please make the necessary adjustments.